### PR TITLE
Add blog entry for Kafka 3.9

### PR DESCRIPTION
**Pull request by cmccabe**

### CODE REVIEW

**Review by mumrah — APPROVED** [review]
LGTM

File: blog.html [hunk 1]
```diff
@@ -22,6 +22,122 @@
...
+                    </li>
+                </ul>
+
+                <h3>Kafka Streams</h3>
+                <p>There are several Kafka Streams improvements in Apache Kafka 3.9.</p>
+                <ul>
+                    <li><b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-1049%3A+Add+config+log.summary.interval.ms+to+Kafka+Streams">KIP-1049: Add config log.summary.interval.ms to Kafka Streams</a>:
+                        </b><br>Introduce the log.summary.interval.ms to control the frequency of summary logs.
+                    </li>
+                    <li><b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-1040%3A+Improve+handling+of+nullable+values+in+InsertField%2C+ExtractField%2C+and+other+transformations">KIP-1040: Improve handling of nullable values in InsertField, ExtractField, and other transformations</a>:
```
Comment (mimaison) [inline blog.html]: That's a Connect feature

File: blog.html [hunk 2]
```diff
@@ -22,6 +22,122 @@
...
+                    <li><b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-1031%3A+Control+offset+translation+in+MirrorSourceConnector">KIP-1031: Control offset translation in MirrorSourceConnector</a>:
+                        </b><br>Add the emit.offset-syncs.enabled configuration, which can be used to disable configuration synchronization.
+                    </li>
+                    <li><b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-1017%3A+Health+check+endpoint+for+Kafka+Connect">KIP-1017: Health check endpoint for Kafka Connect</a>:
+                        </b><br>Add a REST endpoint that can be used to determine if Kafka Connect workers are healthy.
+                    </li>
+                </ul>
+
+                <h3>Conclusion</h3>
+                <p>I hope that this post has given you a sense of all the exciting things that are going on in Kafka 3.9. For more details, take a look at the <a href="https://archive.apache.org/dist/kafka/3.7.0/RELEASE_NOTES.html">release notes</a>, or simply <a href="https://kafka.apache.org/downloads">download</a> the release for yourself.
```
Comment (satishd) [inline blog.html]: `release notes` is pointing to 3.7.0 release, needs to be updated with 3.9.0 link.

**Review by satishd — COMMENTED** [review]
Thanks @cmccabe for the PR, left a minor comment.

**Pull request merged**